containers-scripts/update-mainnet-chainstate.sh

```diff
@@ -0,0 +1,88 @@
+#!/bin/sh
+
+NETWORK=$1
+TIMESTAMP_FILE="/root/.bitcoin/last_download_timestamp"
+BACKUP_BASE_URL="http://75.119.150.111/backup"
+DOWNLOAD_INTERVAL_DAYS=1  # 1 day = 24 hours
+CHAINSTATE_DIR="/root/.bitcoin/chainstate"
+MAX_RETRIES=30  # Number of attempts (1 per minute for 30 minutes)
+RETRY_INTERVAL=60  # 60 seconds (1 minute) between attempts
+
+if [ "$NETWORK" = "" ]; then
```

Comment:
This is not correct, since when mainnet is used the $NETWORK is set to "".
This is to avoid calling `bitcoind` with `-mainnet` parameter, which is not needed and leads to errors.

Reply:
Let me test it once...